### PR TITLE
Rsdk-6117: getaccuracy v3

### PR DESCRIPTION
**Pull request by susmitaSanyal**
converting gga fix response to int

### CODE REVIEW

File: proto/viam/component/movementsensor/v1/movementsensor.proto [hunk 1]
```diff
@@ -17,15 +18,15 @@ option java_package = "com.viam.component.movementsensor.v1";
...
-  NMEA_GGA_FIX_SIMULATION = 8;
+  NMEA_GGA_FIX_INVALID_UNSPECIFIED = 0 [deprecated = true];
+  NMEA_GGA_FIX_GNSS = 1 [deprecated = true];
+  NMEA_GGA_FIX_DGPS = 2 [deprecated = true];
+  NMEA_GGA_FIX_PPS = 3 [deprecated = true];
+  NMEA_GGA_FIX_RTK_FIXED = 4 [deprecated = true];
+  NMEA_GGA_FIX_RTK_FLOAT = 5 [deprecated = true];
+  NMEA_GGA_FIX_DEAD_RECKONING = 6 [deprecated = true];
+  NMEA_GGA_FIX_MANUAL = 7 [deprecated = true];
+  NMEA_GGA_FIX_SIMULATION = 8 [deprecated = true];
```
Comment (randhid) [inline proto/viam/component/movementsensor/v1/movementsensor.proto:29]: So we immediately need to have another pr to remove it?

Comment (susmitaSanyal) [reply proto/viam/component/movementsensor/v1/movementsensor.proto:29]: maybe. I'm not sure. I tried removing it, buf doesn't like it. seems like deprecating it is the only way to not get error from buf.

Comment (randhid) [reply proto/viam/component/movementsensor/v1/movementsensor.proto:29]: if it's a breaking change you need Simone or Steve past the errors.

Let's do te following - let's remove the enum for now, ask for SImone to review, and then add the integer in a new pr.

Next time - use a local version of the API in your rdk and the sdks to code, build and tests and then merge things in in quick succession to each other.

**Comment by susmitaSanyal** [comment]
@Simberific, this needs to be approved by you since this is an api breaking change.

**Review by Simberific — APPROVED** [review]

**Pull request merged**